src/collections/blog/2022/2022-05-10-dockercon-22-hashicorp-talk/index.mdx

```diff
@@ -115,7 +115,8 @@ MeshMap is the world's only visual designer for Kubernetes and all cloud native
 <iframe src="https://www.youtube.com/embed/3DPZafR8VWM" loading="lazy" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen ></iframe>
 </div>
 
-<p><strong>Lee Calcote and Nic Jackson packed a great deal of information in this talk. Find the recording below. The Meshery Extension is now out! Try now, and Share your Experience </strong>
+<p>
```

Comment:
try to exclude these unnecessary line changes

Reply:
For these type of changes I am referring to latest repo of Layer5 and then making the changes according to it. I will avoid them from now.